syntax = "proto3";
package module.protocol;

import "message.proto";

message UserQuery {
  fixed64 user_id = 1;
}

message UserInfo {
  fixed64              user_id = 1;
  int32           last_modified= 2; // 用户最后活跃时间
  repeated KeyStr    subscribe = 3;
  repeated KeyStr      dislike = 4;
  repeated KeyTime      readed = 5;
  repeated KeyTime recommended = 6;
}

message ItemQuery {
  fixed64 item_id = 1;
}

